bvt.py

```diff
@@ -59,8 +59,10 @@ def parse_args_or_exit(argv=None):
 
 if __name__ == "__main__":
     args = parse_args_or_exit(sys.argv[1:])
+    build_status = is_build_action_ok("trunk-ring3", "xe-phase-1-build")
     bvt_status = is_build_action_ok("trunk-ring3", "xe-phase-1-test-ring3")
     if args.dry_run:
```

Comment:
Ah yes, missed that! Don't know what happened to the permissions, but I'll change it back.